docs/jwst.rst

```diff
@@ -54,14 +54,31 @@ All classes share some common attributes:
     available modes well enough to be aware if they are trying to
     simulate an inconsistent or physically impossible configuration. 
 
+Optical Telescope Element (OTE)
+===============================
+
+The JWST Optical Telescope Element consists of the telescope optics that serve all the science instruments and the fine guidance sensor. Most notably, this means the primary, secondary, and tertiary mirrors. The OTE contributes to the overall wavefront error (and therefore the aberrations in instrument PSFs) in a few ways:
+
+  * The limits of precisely manufacturing the mirrors introduce tiny high spatial frequency bumps and ripples of optical path difference
+  * During commissioning, the telescope mirror segments will be aligned and phased as precisely as possible, but small errors in the final aligned configuration will still contribute to WFE
+  * The WFE will vary with field position, which is inherent in the OTE optical design even if perfectly aligned
+  * Aberrations can be introduced by pupil shear or other misalignments between the OTE and each science instrument
+
+Let none of this make you think the JWST optics are deficient in some way! JWST is an extremely precisely engineered optical system. We nevertheless must model these small effects in order to provide users with the most realistic PSF we can.
+
+These effects are simulated at high fidelity in models maintained by Ball Aerospace, which in turn were used to create the OPD map files for the JWST instruments included in WebbPSF.
+
+For each science instrument, if you print ``inst.opd_list`` (where ``inst`` is an instance of an instrument model), you will see a "predicted" OPD map and a "requirements" OPD map. These maps include the OTE contributions to wavefront error listed above. The "requirements" file is more conservative, and includes a realization of the OPD map based on the minimum required performance for JWST. The "predicted" OPD map is informed by ground testing of the true performance of the SIs and OTE, and is a better approximation of what we think the true performance will be.
+
+Of course, these OPD maps only simulate the OTE contribution to the overall wavefront error. Additional details on the SI-specific wavefront error models are given under each instrument model section.
 
```

Comment:
> only simulate the OTE contribution

Again, not quite. it's more like "The OTE contribution, OTE to ISIM misalignments, and estimates for stability and measurement uncertainties in both OTE and ISIM"